### ISSUE TITLE
Single jar distribution 

### ISSUE THREAD

**Issue by pun-ky** [Cognifide/cogboard#323]
Right now running CB is available easily only when using Docker. However Docker in some cases could not be available. What is more I see that Cogboard right now requires running 2 separate web servers. Netty/Knotx and nginx. I am aware that this approach might have some benefits however it could simply make this whole solution more complex that it is necessary. How about serving frontend app via Netty too and creating all-in-one self-extractable jar to provide much easier quickstart to be able to 
Increase general tool availability.

In other words, that Docker could make afraid some potential users completely unnecessarily 😉

**Comment by pun-ky** [comment]
It is just more probable that target server is having already installed Java than Docker...

**Comment by szymon-owczarzak** [comment]
@pun-ky thank You for your feedback - we will take a look at it when we have some resources.

**Comment by pun-ky** [comment]
I mean... configure distribution to be the same as in Knotx Starterkit. It outputs both distro ZIP and single Docker image.
